srv/admin-service.cds

```diff
@@ -1,28 +1,45 @@
-using { sap.common.Languages as CommonLanguages } from '@sap/cds/common';
-using { my.bookshop as my } from '../db/schema';
+using
```

Comment:
I think this file still uses the old formatting. Can we reformat it with the new settings?